go/client/chat_svc_handler.go

```diff
@@ -159,6 +160,9 @@ func (c *chatServiceHandler) ReadV1(ctx context.Context, opts readOptionsV1) Rep
 		Offline:          threadView.Offline,
 		IdentifyFailures: threadView.IdentifyFailures,
 	}
+	if opts.Pagination != nil {
```

Comment:
Do we need this `if`?

Reply:
If we're returning all the message back, any pagination info we return would be unneeded, right?

Reply:
I think we always want the pagination, even if we didn't use it on the request. That way, in case the next requests needs it, it will be there.

Reply:
Hmm, removed the if statement, although I don't quite understand how it'd be used on a non-paginated request. I guess perhaps if you get all messages, but then you want to fetch only newer messages?

Reply:
Yeah that is a valid one, you could envision something that just spams the `prev` page looking for new stuff.